primitives/io/src/lib.rs

```diff
@@ -554,7 +554,18 @@ pub trait Hashing {
 	}
 }
 
+/// Interface that provides functions to access the offchain database access.
+#[runtime_interface]
+pub trait OffchainIndex {
+	/// Write a key value pair to the offchain worker database in a buffered fashion.
+	fn write_kv(&mut self, key : &[u8], value : &[u8]) {
+		self.local_ocw_storage_write_kv(key,value)
+	}
+}
```

Comment:
for completeness a `remove` function would be good as well

Reply:
That would require a few adjustments, so it might make more sense to handle that in a separate PR?

Reply:
Sure thing